### ISSUE TITLE
Add keyboard navigation for expandable sections

### ISSUE THREAD

**Issue by cdriehuys** [cdriehuys/chmvh-website#60]
Expandable selections should be focusable by keyboard navigation so that the sections can be expanded/collapsed without the use of the mouse.
